unique-universes/src/exts/minigames.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+import random
+from typing import TYPE_CHECKING
+
+import disnake
+from disnake.ext import commands
+from src.views.shoot import ShootMenu
+
+if TYPE_CHECKING:
+    from src.bot import Universe
+    from src.views.shoot import ShootStats
+
+
+__all__: tuple[str, ...] = (
+    "generate_random_stats",
+    "Minigames",
+)
+
+
+def _generate_fake_mac() -> str:
+    tokens = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+    seq = random.choices(tokens, k=16)  # noqa: S311
+    return ":".join([f"{seq[i]}{seq[i+1]}" for i in range(0, len(seq), 2)])
```

Comment:
You could instead do `random.choices` with k=2 8 times and join that. Seems more readable that way